src/intTest/resources/outbound_uat_data/acceptance/4_2.type4-immigrant-mandatory.edifact.dat

```diff
@@ -0,0 +1,20 @@
+UNB+UNOA:2+TES5+XX11+200610:1438+00000001'
+UNH+00000001+FHSREG:0:1:FH:FHS001'
+BGM+++507'
+NAD+FHS+XX1:954'
+DTM+956:19920115:102'
+RFF+950:G1'
+S01+1'
+RFF+TN:1'
+NAD+GP+2750922,295:900'
+HEA+ACD+A:ZZZ'
+HEA+ATP+4:ZZZ'
+DTM+137:202006101438:203'
```

Comment:
This is the acceptance date is should be provided by the acceptanceDate FHIR parameter